src/k8s/vfs/json-locator.ts

```diff
@@ -0,0 +1,248 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { MappingItem, Node, NodeProvider } from './locator-util';
+
+const parse = require('json-to-ast');
```

Comment:
please use `import` instead of require, it should work just as well

Reply:
When using import it couldn't be built for some reason. I'll check it again.